src/to_square_example.c

```diff
@@ -0,0 +1,177 @@
+#include <pebble.h>
+
+
+#define ABS(a) (((a) > 0) ? (a) : -1 * (a))
+#define IMAGES_LENGTH 2
+
+
+
+static Window *s_main_window;
+static Layer *icon_layer;
+static GDrawCommandImage* images [IMAGES_LENGTH];
```

Comment:
The `*` should be next to the variable name, not the type name. Also remove spaces between array names and lengths. 

`static GDrawCommandImage *images[IMAGES_LENGTH];`